src/main/java/gregtech/integration/exnihilo/metatileentities/MetaTileEntitySieve.java

```diff
@@ -0,0 +1,62 @@
+package gregtech.integration.exnihilo.metatileentities;
+
+import gregtech.api.gui.GuiTextures;
+import gregtech.api.gui.ModularUI;
+import gregtech.api.gui.widgets.CycleButtonWidget;
+import gregtech.api.gui.widgets.ImageWidget;
+import gregtech.api.gui.widgets.ProgressWidget;
+import gregtech.api.gui.widgets.ToggleButtonWidget;
+import gregtech.api.metatileentity.MetaTileEntity;
+import gregtech.api.metatileentity.SimpleMachineMetaTileEntity;
+import gregtech.api.metatileentity.interfaces.IGregTechTileEntity;
+import gregtech.client.renderer.texture.Textures;
+import gregtech.integration.exnihilo.ExNihiloModule;
+
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.util.ResourceLocation;
+
+import org.jetbrains.annotations.NotNull;
+
+public class MetaTileEntitySieve extends SimpleMachineMetaTileEntity {
```

Comment:
Why is this subclass needed?

Reply:
Similar to above with the JEI preview, it'll just look like this without the subclass
<img width="468" height="411" alt="image" src="https://github.com/user-attachments/assets/2fc0cfb7-eb93-48bb-bf35-dbc805926a40" />